src/components/ActionsMenu/Item/ActionsMenuItem.st.css

```diff
@@ -0,0 +1,73 @@
+:import {
```

Comment:
It looks like the styles doesn't completely match zeroheight, can you please notify me once you are done with the styling?